lib/ansiblelint/rules/IncludeMissingFileRule.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2016, Will Thames and contributors
```

Comment:
These copyright lines are invalid

Reply:
Should it be `# Copyright (c) 2020, Will Thames and contributors` then?

Reply:
No, it's a new module, all code is yours. You can write yourself and Ansible Project.

Reply:
Is this ok then?
```
# Copyright (c) 2020, Joachim Lusiardi
# Copyright (c) 2020, Ansible Project
```

Reply:
Yes. Usually, the copyright states when the content was created/updated.